src/main/scala/org/apache/spark/shuffle/ucx/rpc/UcxDriverRpcEndpoint.scala

```diff
@@ -81,6 +91,7 @@ class UcxDriverRpcEndpoint(override val rpcEnv: RpcEnv) extends ThreadSafeRpcEnd
       executorToDpuAddress += executorId -> dpuSockAddress
 
       nvkvLock.getOrElseUpdate(SerializationUtils.deserializeInetAddress(dpuSockAddress.value), {0})
+      executerLocalIdMap.getOrElseUpdate(SerializationUtils.deserializeInetAddress(dpuSockAddress.value), {0})
```

Comment:
Here the key is DPU sock addr. but in other places above seems the key is the executor address (?)

Reply:
Actually I always use the local DPU address to differ between executers from the same node because it's already passed to the driver for advertising purposes.
I'm not using the host address at all. Check `receiveAndReply`, But you are right, in some place I declared vars with the name `execAdd` and it's wrong, need to change it to `execDpuAdd` or similar